source/vibe/stream/openssl.d

```diff
@@ -469,6 +471,17 @@ final class OpenSSLContext : SSLContext {
 			SSL_CTX_set_cipher_list(m_ctx, toStringz(list));
 	}
 
+	/** Make up a context ID to assign to the SSL context.
+
+		This is very required when doing client cert authentication,
+		otherwise many connections will go aborted.
+	*/
+	void setContextID()
```

Comment:
I wanted to pick a sane default so it will Just Work if you don't care about the particular value. It has to be unique within a group of servers behind a load balancer, so the protocol can tell it doesn't have a cached session on that particular server and do the handshake again.

BUT, it also has to be configurable for client code in case someone wants to go to the trouble to configure a session pool. Apparently, "tickets" will be better in any case, but not necessarily supported by every client and adding a lot more complexity. 

You're right about the name change, will fix.